schema/codelists/contractStatus.csv

```diff
@@ -1,7 +1,9 @@
-Code,Title,Description
-pending,Pending,"This contract has been proposed, but is not yet in force. It might be awaiting signature."
-active,Active,"This contract has been concluded by all the parties, and is now legally in force."
-cancelled,Cancelled,This contract has been cancelled prior to being concluded.
-terminated,Terminated,"This contract was concluded and in force, and has now come to a close. This might be due to the successful completion of the contract, or might be early termination due to some non-completion."
-terminatedEarly,Terminated early,"This contract was concluded and in force, and is terminated early due to some non-completion."
-terminatedSuccessfully,Terminated successfuly,"This contract was concluded and in force, and is terminated due to its successful completion."
+Code,Title,Description,Deprecated,Deprecation note
+pending,Pending,"This contract has been proposed, but is not yet in force. It might be awaiting signature.",1.2,
+active,Active,"The award has been made and the standstill period, if any, has ended. The contract has not yet terminated.",,
+terminated,Terminated,"The contract was concluded and in force, and has now come to a close. This might be due to the successful completion of the contract, or might be early termination due to some non-completion.",,
+terminatedSuccessfully,Terminated successfully,"The contract was concluded and in force, and is terminated due to its successful completion.",,
+terminatedEarly,Terminated early,"The contract was concluded and in force, and is terminated early due to some non-completion.",,
+cancelled,Cancelled,"The contracting process has been cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the award has been made and the standstill period, if any, has ended.",,
```

Comment:
```suggestion
cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the award was made and the standstill period, if any, had ended.",,
```